packages/iov-bns/src/decode.ts

```diff
@@ -156,38 +158,51 @@ function parseRegisterBlockchainTx(
 ): RegisterBlockchainTx {
   const id = ensure(msg.id, "id");
   const details = ensure(msg.details, "details");
+
+  const chain = ensure(details.chain, "details.chain");
```

Comment:
Are all of these required and guaranteed to be present by the backend?

Good use of ensure.

Reply:
Can we trust the `gogoproto.nullable` annotations in https://github.com/iov-one/weave/blob/master/x/nft/blockchain/codec.proto#L41 / https://github.com/iov-one/weave/blob/master/x/nft/blockchain/codec.proto#L15?